### PR TITLE
feat(demo): update Section page

### PR DESCRIPTION
**Pull request by aiheon**
### Proposed Changes

https://coveord.atlassian.net/browse/UITOOL-538

### Potential Breaking Changes

none

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/website/src/pages/layout/Section.tsx [hunk 1]
```diff
@@ -1,241 +1,42 @@
...
-                                        <Label>Font Family</Label>
-                                    </Input>
-                                </LabeledInput>
-                            </Section>
-                        </Section>
-                    </Section>
-                </Form>
+    export default () => (
+        <Section title="Section title" description="Section description.">
+            <Section title="Section title inside the anoter Section">
```
Comment (GermainBergeron) [inline packages/website/src/pages/layout/Section.tsx:10]: Maybe remove this one from the basic demo and add a new one for nested sections?

Comment (GermainBergeron) [reply packages/website/src/pages/layout/Section.tsx:10]: It doesn't really look like a child without the level=2 prop, also the child doesn't look weird to me 😛 

image

Comment (aiheon) [reply packages/website/src/pages/layout/Section.tsx:10]: :joy:  It's looking weird on my cheap monitor :see_no_evil:

**Review by GermainBergeron — APPROVED** [review]

File: packages/react/src/components/section/Section.tsx [hunk 2]
```diff
@@ -4,10 +4,27 @@ import * as React from 'react';
...
+    /**
+     * The text or custom JSX content to the title of the Section
+     */
     title?: React.ReactNode;
+    /**
+     * The text or custom JSX content to the description of the Section
+     */
     description?: React.ReactNode;
+    /**
+     * Additionnal CSS class to stle the Section
```
Comment (gdostie) [inline packages/react/src/components/section/Section.tsx:16]:
```suggestion
     * Additionnal CSS class to set on the Section
```

File: packages/react/src/components/section/Section.tsx [hunk 3]
```diff
@@ -4,10 +4,27 @@ import * as React from 'react';
...
+     */
     className?: string;
+    /**
+     * Visual modifiers to apply on the component
+     */
     mods?: SectionMods | SectionMods[];
+    /**
+     * Determine the style of the Section
```
Comment (gdostie) [inline packages/react/src/components/section/Section.tsx:24]:
```suggestion
     * Determines the level of importance of the section. Smaller number means bigger title, think of it as heading levels.
```

**Review by gdostie — APPROVED** [review]

**Pull request merged**